scripts/emulate_app.sh

```diff
@@ -177,35 +178,43 @@ if [ ! -d "./android/app" ]; then
     echo "Try './scripts/$(basename $0)' from zingo-mobile root directory." >&2
     exit 1
 fi
-cd android
 
-echo -e "\nInstalling latest build tools, platform tools, and platform..."
-sdkmanager --install 'build-tools;33.0.2' platform-tools
+cd android
+# Kill all emulators
+../scripts/kill_emulators.sh
 
-echo "Installing latest emulator..."
-sdkmanager --install emulator --channel=0
+test_report_dir="app/build/outputs/emulate_app_reports/${abi}"
+rm -rf "${test_report_dir}"
+mkdir -p "${test_report_dir}"
 
-echo "Installing system image..."
 avd_name="${avd_skin}-android-${api_level}_${api_target}_${arch}"
 sdk="system-images;android-${api_level};${api_target};${arch}"
-sdkmanager --install "${sdk}"
-sdkmanager --licenses
-
-# Kill all emulators
-../scripts/kill_emulators.sh
+platform="platforms;android-${api_level}"
 
 if [[ $create_snapshot == true ]]; then
+    sdkmanager --licenses
```

Comment:
i don't think it looks better how the daemon starts running and it prints "All licenses accepted" as the first thing when you run the script. also, all the examples show accepting licenses after the system images. can we put this back to where it was (after installing system images)?

Reply:
guess this is a little confusing to me... doesnt it require is to accept the licenses to install the system images?

Reply:
that would make sense i was just following the "norm" but we can leave it here if you prefer